src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -230,15 +219,16 @@ function BaseSelectionList({
     };
 
     const selectAllRow = () => {
-        onSelectAll();
+        if (onSelectAll) {
+            onSelectAll();
+        }
         if (shouldShowTextInput && shouldPreventDefaultFocusOnSelectRow && textInputRef.current) {
             textInputRef.current.focus();
         }
     };
 
-    const selectFocusedOption = (e) => {
-        const focusedItemKey = lodashGet(e, ['target', 'attributes', 'id', 'value']);
-        const focusedOption = focusedItemKey ? _.find(flattenedSections.allOptions, (option) => option.keyForList === focusedItemKey) : flattenedSections.allOptions[focusedIndex];
+    const selectFocusedOption = () => {
+        const focusedOption = flattenedSections.allOptions[focusedIndex];
```

Comment:
Why was this removed? 
```ts
       const focusedItemKey = lodashGet(e, ['target', 'attributes', 'id', 'value']);
        const focusedOption = focusedItemKey ? _.find(flattenedSections.allOptions, (option) => option.keyForList === focusedItemKey) : flattenedSections.allOptions[focusedIndex];
```

Reply:
It was always undefined as property id does not exist on attributes